src/parser/csql_grammar.y

```diff
@@ -28466,6 +28466,28 @@ pt_check_one_stmt(char* p)
                     }
                 }
           }
+        else if((p[0] == '-' && p[1] == '-') || (p[0] == '/' && p[1] == '/'))
+        {
+                for(p += 2; *p ; p++)
+                {
+                    if (*p == '\n')
```

Comment:
Instead of using a for or if statement,
```
p = strchr(p, '\n');
```
How about using ?

Reply:
I'd like to keep the current code in case something like this happens:

```
SELECT * FROM dblink('192.168.1.8:55300:demodb:dba::',
'select 1, 'a' -- test ') as rt(col1 int);
```